account_analytic_project/model/account_analytic_account.py

```diff
@@ -0,0 +1,77 @@
+# -*- encoding: utf-8 -*-
+"""Extend account.analytic.account."""
```

Comment:
Requirement in your dev env? In all your code, this kind of comment appears only twice. I'll not block for this comment and you already have my :+1: . It's just a matter of consistency and IMO this kind of comment doesn't provide an added value.